src/api/posts/src/bin/article-error.js

```diff
@@ -0,0 +1,13 @@
+class ArticleError extends Error {
```

Comment:
Let's get rid of the `bin` directory completely, everything can live in src.

Reply:
Done